source/community/release_notes/sumac/dev_op_release_notes.rst

```diff
@@ -17,20 +17,56 @@ To view the end-user facing docs, see the :doc:`feature_release_notes`.
+- `Course assets are now served by a view rather than a middleware <https://github.com/openedx/edx-platform/issues/34702>`_
+   - Background: The LMS/CMS previously handled course asset requests (asset-v1: andc4x URLs) via a middleware called StaticContentServer. This middleware has been converted to a view.
+   - Action: If your deployment has a custom MIDDLEWARE list in Django, you will need to remove this item at the time of upgrade to Sumac. Otherwise, no action is needed.
+- Ubuntu 22.04 Related Operators Note
+   - `PR <https://github.com/openedx/edx-platform/pull/35450>`_
+   - In newer versions of ubuntu the MD4 hashing algorithm is disabled by default. To enable it the openssl config needs to be updated in a manner similar to what's being done here. Alternatively, you can set the FEATURES['ENABLE_BLAKE2B_HASHING'] setting to True which will switch to a newer hashing algorithm where MD4 was previously used.
+      Because this hashing is being used as a part of the edx-platform caching mechanism, this will effectively clear the cache for the items that use this hash. The will impact any items where the cache key might have been too big to store in memcache so it's hard to predict exactly which items will be impacted.
+- Added override options to commerce related CTA URLs in edx-platform
+   - Background: Extension points have been added have been added to commerce app in: `PR1 <https://github.com/openedx/edx-platform/pull/35441>`_, `PR2 <https://github.com/openedx/edx-platform/pull/35501>`_ so Open edX community members who wants to extend the commerce functionality can do so without explicitly adding code into edx-platform codebase. For more information into extension points see Pluggable override section in `Extension Points <https://github.com/openedx/edx-platform/blob/master/docs/concepts/extension_points.rst>`_.
+   - Additional Considerations: Commerce app itself is slated for deprecation nevertheless.
+- `courseware.enable_navigation_sidebar <https://github.com/openedx/edx-platform/blob/38f73442e78a8b9afb5543facd170dca830acb1a/lms/djangoapps/courseware/toggles.py#L86>`_  is now enabled by default. It was disabled by default in Redwood.
+
+- New Forums Implementation (migration from Ruby to Python forums backend)
+
+   - For users running Tutor, they will automatically switch to the new app. Instructions to migrate data from mongodb to mysql is available in the plugin README https://github.com/overhangio/tutor-forum/ (not yet merged, see `feat: upgrade to sumac <https://github.com/overhangio/tutor-forum/pull/49/files>`_).
+
+   - Users not running Tutor will by default remain on the legacy `cs_comments_service` app. They need to read this part of the forum app README: https://github.com/openedx/forum?tab=readme-ov-file#forum-v2-toggle Some settings need to be defined even for legacy app users.
+
+
 Settings and Toggles
```

Comment:
I'm not clear on what this is a list of. Are these all new settings and toggles? "new and updated" ? 

Were there any settings and toggles removed?

Reply:
I don't understand the question. This was the list of new/updated toggles @mariajgrimaldi was kind enough to help me generate.

There were a TON of removals as documented in https://github.com/openedx/wg-build-test-release/issues/398:
```
COURSEGRAPH_DUMP_COURSE_ON_PUBLISH
FEATURESENABLE_LIBRARY_AUTHORING_MICROFRONTEND
FEATURESENABLE_V2_CERT_DISPLAY_SETTINGS
commerce.transition_to_coordinator.checkout
commerce.transition_to_coordinator.refund
contentstore.library_authoring_mfe
discussions.enable_reported_content_notifications
learner_dashboard.enable_b2c_subscriptions
notifications.enable_coursewide_notifications
notifications.enable_notifications_filters
notifications.enable_ora_staff_notifications
notifications.show_notifications_tray
studio.enable_course_update_notifications
BLOCKSTORE_BUNDLE_CACHE_TIMEOUT
BUNDLE_ASSET_STORAGE_SETTINGS
BUNDLE_ASSET_URL_STORAGE_KEY
BUNDLE_ASSET_URL_STORAGE_SECRET
BLOCKSTORE_BUNDLE_CACHE_TIMEOUT
COURSEGRAPH_CONNECTION
COURSEGRAPH_JOB_QUEUE
```

In that issue I asked if anyone thought any of these needed explanation.

Do you think they should be included in a separate Toggles/settings removed" section?

Happy to do whatever works. Fighting to nail jell-o to a tree here.

Reply:
I would love to see that list of removed toggles included as a separate section.

Reply:
Change is waiting on a ReadTheDocs build.